Resources/Prototypes/DeltaV/Entities/Objects/Specific/Justice/trialtimer.yml

```diff
@@ -0,0 +1,22 @@
+- type: entity
+  id: TrialTimer
+  parent: SignalTimer
+  name: trial timer
+  description: A fancy timer with a screen, designed for use in court rooms.
```

Comment:
This should probably have some sort of reference to Trials only supposed to be 10 minutes long under Standard Operating Procedure.